src/test/java/org/opensearch/security/privileges/RestLayerPrivilegesEvaluatorTest.java

```diff
@@ -95,28 +102,45 @@ public void testEvaluate_Initialized_Success() {
 
         PrivilegesEvaluatorResponse response = privilegesEvaluator.evaluate(TEST_USER, Set.of(action));
 
-        assertNotNull(response);
-        assertFalse(response.isAllowed());
-        assertFalse(response.getMissingPrivileges().isEmpty());
-        assertTrue(response.getResolvedSecurityRoles().isEmpty());
+        assertThat(response.isAllowed(), equalTo(false));
+        assertThat(response.getMissingPrivileges(), equalTo(Set.of(action)));
+        assertThat(response.getResolvedSecurityRoles(), Matchers.empty());
         verify(configModel, times(3)).getSecurityRoles();
     }
 
-    @Test(expected = OpenSearchSecurityException.class)
-    public void testEvaluate_NotInitialized_ExceptionThrown() throws Exception {
-        String action = "action";
-        privilegesEvaluator.evaluate(TEST_USER, Set.of(action));
+    @Test
+    public void testEvaluate_NotInitialized_NullModel_ExceptionThrown() throws Exception {
+        // Null out the config model
+        privilegesEvaluator.onConfigModelChanged(null);
+        final OpenSearchSecurityException exception = assertThrows(
+            OpenSearchSecurityException.class,
+            () -> privilegesEvaluator.evaluate(TEST_USER, null)
+        );
+        assertThat(exception.getMessage(), equalTo("OpenSearch Security is not initialized."));
+        verify(configModel, never()).getSecurityRoles();
+    }
+
+    @Test
+    public void testEvaluate_NotInitialized_NoSecurityRoles_ExceptionThrown() throws Exception {
+        final OpenSearchSecurityException exception = assertThrows(
+            OpenSearchSecurityException.class,
+            () -> privilegesEvaluator.evaluate(TEST_USER, null)
+        );
+        assertThat(exception.getMessage(), equalTo("OpenSearch Security is not initialized."));
+        verify(configModel).getSecurityRoles();
     }
 
     @Test
```

Comment:
I think I get you an error in my pull request, let me know if you need a hand addressing it.

```
Suite: Test class org.opensearch.security.privileges.RestLayerPrivilegesEvaluatorTest
  2> org.mockito.exceptions.misusing.InvalidUseOfMatchersException: 
    Invalid use of argument matchers!
    2 matchers expected, 1 recorded:
    -> at org.opensearch.security.privileges.RestLayerPrivilegesEvaluatorTest.testMapRoles_ReturnsMappedRoles(RestLayerPrivilegesEvaluatorTest.java:137)

    This exception may occur if matchers are combined with raw values:
        //incorrect:
        someMethod(any(), "raw String");
    When using matchers, all arguments have to be provided by matchers.
    For example:
        //correct:
        someMethod(any(), eq("String by matcher"));

    For more info see javadoc for Matchers class.
        at app//org.opensearch.security.privileges.RestLayerPrivilegesEvaluatorTest.testMapRoles_ReturnsMappedRoles(RestLayerPrivilegesEvaluatorTest.java:137)

Tests with failures:
 - org.opensearch.security.privileges.RestLayerPrivilegesEvaluatorTest.testMapRoles_ReturnsMappedRoles
```

Reply:
No worries, thank you for your help. I will take a look at the failure now.